raiden/network/proxies/token_network_registry.py

```diff
@@ -128,6 +128,10 @@ def add_token(
         except BadFunctionCallOutput:
             raise_on_call_returned_empty(block_identifier)
         else:
+            if token_address == NULL_ADDRESS_BYTES:
+                raise BrokenPreconditionError(
+                    "The call to register a token at 0x00..00 will fail."
+                )
```

Comment:
This check should go to line 118 below `if block_identifier == "latest":` and raise a `ValueError`. Because in the `try` section, we already use `token_address` so if it's set to null address bytes then the value is invalid.